src/main/java/trackitnus/commons/core/Messages.java

```diff
@@ -16,6 +16,7 @@
     public static final String MESSAGE_INVALID_LESSON_DISPLAYED_INDEX = "The lesson index provided is invalid";
 
     public static final String MESSAGE_INVALID_MODULE_DISPLAYED_INDEX = "The module index provided is invalid";
-    public static final String MESSAGE_MODULE_DOES_NOT_EXIST = "There is no module that matches the given module code";
+    public static final String MESSAGE_MODULE_DOES_NOT_EXIST = "The module specified in the command doesn't exist.";
```

Comment:
Out of curiosity, why change this? I don't see anything wrong the old version

Reply:
Actually I changed this so that it can be used for cases of module not exist in Lessons. The original version is only good for delete module